lib/install/data/data.go

```diff
@@ -256,5 +256,12 @@ func (d *Data) CopyNonEmpty(src *Data) error {
 
 	d.Timeout = src.Timeout
 
+	if src.DNS != nil {
```

Comment:
The DNS in data is []net.IP so it doesn't have an IsSet field. I am trying to not change existing data structure within the data struct. So I used this logic to enable reset for dns.